### PR TITLE
Add German language support

### PR DESCRIPTION
**Pull request by 6R1M5L07H**
German translation file created

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-version: "3.8"
```
Comment (ilyarolf) [inline docker-compose.yml:1]: Do not add any changes to the docker compose file.

File: docker-compose.yml [hunk 2]
```diff
@@ -24,7 +23,7 @@ services:
       - 4040
       - 5000 # ${WEBAPP_PORT}
     volumes:
-      - /AiogramShopBot:/bot/data
+      - ./AiogramShopBot:/bot/data
```
Comment (ilyarolf) [inline docker-compose.yml:26]: Do not add any changes to the docker compose file.

**Review by ilyarolf — CHANGES_REQUESTED** [review]
Do not add any changes to the docker compose file.

File: docker-compose.yml (hunk 1 above)
Comment (6R1M5L07H) [reply docker-compose.yml:1]: ['version' is obsolete](https://docs.docker.com/reference/compose-file/version-and-name/), but I can revert is, when you wish

File: docker-compose.yml (hunk 2 above)
Comment (6R1M5L07H) [reply docker-compose.yml:26]: reverted

**Review by ilyarolf — APPROVED** [review]

**Pull request merged**